src/clr-angular/layout/tabs/providers/tabs.service.ts

```diff
@@ -26,7 +28,7 @@ export class TabsService {
 
   get overflowTabs() {
     return this.children.filter((tab: ClrTab) => {
-      return tab.tabLink.inOverflow === true;
+      return this.orientation !== 'vertical' && tab.tabLink.inOverflow;
```

Comment:
I know that previously we didn't have a spec for this service. Would you mind adding a spec for this service that tests the two changes here?

Reply:
Will add a test!

Reply:
On a second check, there is a reason not to have a spec for this service. To be properly initialized it will requires a list of ClrTab items. This can be done most easily with a dummy test component. If we do this here, it will become a duplicate of what we have in tabs.spec.ts.
The service is already tested there, directly and indirectly, so I added my tests there instead of creating a new spec here.
I test the service state changes, and how the overflow is handled in vertical mode. Should be enough.